evals/deepeval_on_swe_bench.py

```
@@ -0,0 +1,174 @@
+
+
+async def run_cognify_base_rag():
+    from cognee.api.v1.add import add
+    from cognee.api.v1.prune import prune
+    from cognee.api.v1.cognify.cognify import cognify
+
+    await prune.prune_system()
+
+    await add("data://test_datasets", "initial_test")
+
+    graph = await cognify("initial_test")
```

Ruff

Local variable `graph` is assigned to but never used

Remove assignment to unused variable `graph`

(F841)